src/unistd.rs

```diff
@@ -1520,7 +1520,7 @@ pub fn getgroups() -> Result<Vec<Gid>> {
     // equal to the value of {NGROUPS_MAX} + 1.
     let ngroups_max = match sysconf(SysconfVar::NGROUPS_MAX) {
         Ok(Some(n)) => (n + 1) as usize,
-        Ok(None) | Err(_) => <usize>::max_value(),
```

Comment:
In this case, the reason the code used to use max_value instead of MAX is that the latter only stabilized in Rust 1.43.0.  max_value is older.

Reply:
Yes, that's right